### ISSUE TITLE
NeuroBEM: Hybrid Aerodynamic Quadrotor Model. (arXiv:2106.08015v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#7945]
NeuroBEM: Hybrid Aerodynamic Quadrotor Model. (arXiv:2106.08015v1 [cs.RO])<br>
https://ift.tt/3vosDJP<br>
<p>Quadrotors are extremely agile, so much in fact, that classic first-principle-models come to their limits. Aerodynamic effects, while insignificant at low speeds, become the dominant model defect during high speeds or agile maneuvers. Accurate modeling is needed to design robust high-performance control systems and enable flying close to the platform's physical limits. We propose a hybrid approach fusing first principles and learning to model quadrotors and their aerodynamic effects with unprecedented accuracy. First principles fail to capture such aerodynamic effects, rendering traditional approaches inaccurate when used for simulation or controller tuning. Data-driven approaches try to capture aerodynamic effects with blackbox modeling, such as neural networks; however, they struggle to robustly generalize to arbitrary flight conditions. Our hybrid approach unifies and outperforms both first-principles blade-element theory and learned residual dynamics. It is evaluated in one of the world's largest motion-capture systems, using autonomous-quadrotor-flight data at speeds up to 65km/h. The resulting model captures the aerodynamic thrust, torques, and parasitic effects with astonishing accuracy, outperforming existing models with 50% reduced prediction errors, and shows strong generalization capabilities beyond the training set.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>

**Comment by Azircarry** [comment]
> NeuroBEM：混合空气动力学四旋翼模型。（arXiv：2106.08015v1 [cs.RO]） https://ift.tt/3vosDJP
> 
> 事实上，四旋翼飞行器非常灵活，以至于经典的第一性原理模型达到了极限。空气动力学效应虽然在低速时微不足道，但在高速或敏捷机动时成为模型的主要缺陷。需要精确的建模来设计强大的高性能控制系统，并使飞行接近平台的物理极限。我们提出了一种混合方法，融合了第一性原理和学习，以前所未有的精度对四旋翼飞行器及其空气动力学效应进行建模。第一性原理无法捕捉到这种空气动力学效应，使传统方法在用于仿真或控制器调整时不准确。数据驱动的方法试图通过黑盒建模（例如神经网络）来捕捉空气动力学效应;然而，他们很难有力地概括到任意的飞行条件。我们的混合方法统一并优于第一性原理叶片元素理论和学习的残差动力学。它是在世界上最大的动作捕捉系统之一中进行的，使用速度高达65公里/小时的自主四旋翼飞行数据进行评估。由此产生的模型以惊人的精度捕获空气动力学推力、扭矩和寄生效应，性能优于现有模型，预测误差降低了 50%，并显示出超越训练集的强大泛化能力。
> 
> 
> 
> 通过 cs.RO 更新 arXiv.org http://arxiv.org/

Hello, I've been reading this article recently and trying to get through the source code, and I'd like to ask you about some of the environments and simulation tools that the source code needs, as well as questions about the order in which the source code runs. Sorry to bother you! I hope you can help me, thank you
